variables.tf

```diff
@@ -14,6 +14,17 @@ variable "service" {
 variable "project" {
   type = string
 }
+variable "data_classification" {
+  description = "Data Classification type. valid values are private, public, internal."
+  type        = string
+  default     = "internal"
```

Comment:
I wonder if this should default to "". (The thinking being it may be better to leave this unset than misclassify.)

Reply:
Changed the default value to `""`